packages/itwinui-css/src/tabs/base.scss

```
@@ -336,4 +335,8 @@ $borderless-horizontal-tab-min-height: calc(
 
 @mixin iui-tabs-content {
   grid-area: tabs-content;
+
+  &[hidden='true'] {
```

there is no `='true'` in the markup
```suggestion
  &[hidden] {
```